docs/src/modules/branding/BrandingRoot.tsx

```diff
@@ -39,6 +39,13 @@ declare module '@material-ui/core/styles/createPalette' {
   interface PaletteOptions extends CustomPaletteOptions {}
 }
 
+// Update the Typography's variant prop options
```

Comment:
Won’t these augmentation affect the whole docs?

Reply:
Yeah, I suspect it does. I haven't removed h5 or subheader1 for this reason. I have noticed we augment the palette too.

Reply:
Won't this create problems with the ts example for the `Typography` component itself? I believe we are documenting all variants there - https://next.material-ui.com/components/typography/#component

Reply:
Do you have a specific problem in mind? I have honestly no idea if it will hit us later on. The CI is green, I wasn't even sure it would be the case 😁

Reply:
If the CI is green then we are fine I guess :) I thought we would break any other usage of `<Typography variant="h5" />` :)

Reply:
You were just removing them by setting them to `undefined`, I thought you are removing them with the module augmentaiton. Sorry my confusion :D